CHANGELOG.rst

```diff
@@ -10,6 +10,9 @@ Unreleased
 New Features
 ~~~~~~~~~~~~
 
+* finagle-core: Recognize `fixedinet` name scheme in global `com.twitter.finagle.Namer`. This
```

Comment:
I got a conflict in the changelog while merging this pr, would you move this section to the `Unreleased` section? Thanks!

Reply:
Done. Please check.